glommio/src/executor/mod.rs

```diff
@@ -938,6 +939,14 @@ impl LocalExecutor {
         };
 
         let io_requirements = IoRequirements::new(latency, index);
+        tracing::trace!(
```

Comment:
wondering if it wouldn't be better to just add `trace` instead of `tracing::trace`? and `use trace` on top? Should make it easier for us to compile-time replace it with a usdt macro in the future.

Reply:
yeah, the usdt stuff is a little trickier so we might have to wrap them both in another macro to be able to do that.. Again, I am fine just removing the "example" tracepoints. They aren't the important part at all. I think I can address all these comments pretty easily.